github.json

```diff
@@ -31632,7 +31632,7 @@
             },
             {
               "name": "mergeMethod",
-              "description": "The merge method to use. If omitted, defaults to 'MERGE'",
+              "description": "the merge method to use. If omitted, defaults to 'MERGE'",
```

Comment:
```suggestion
              "description": "The merge method to use. If omitted, defaults to 'MERGE'",
```

Reply:
Thanks @adi611, the Github.json file still has changes. Could you undo these please? For some reason, it won't let me change them via a code review.

Reply:
> Thanks @adi611, the Github.json file still has changes. Could you undo these please? For some reason, it won't let me change them via a code review.

Done @minimalsm

Reply:
Thanks @adi611! 

Can you check the change requests in the **Files changed** tab? There are still quite a few changes we need to revert here.

Reply:
Is it ok now?

Reply:
Yup, thanks!